pre-aio-node-with-zuul-ref.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+
```

Comment:
Add comments to describe the functions to be implemented in this script and explain the relative input parameters.

Reply:
Will add it